bigbluebutton-web/grails-app/conf/bigbluebutton.properties

```diff
@@ -261,7 +261,7 @@ userCameraCap=3
 maxPinnedCameras=3
 
 # Mute the meeting on start
```

Comment:
```suggestion
# Microphone is muted by default when users start sharing
```

I think that the current description does not reflect the true function of this config.